src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -580,28 +570,22 @@ string Visitor::visit(Parser::PnameNsContext* ctx) {
 }
 
 // ____________________________________________________________________________________
-SparqlQleverVisitor::PrefixMap SparqlQleverVisitor::visit(
-    SparqlAutomaticParser::PrologueContext* ctx) {
+void Visitor::visit(Parser::PrologueContext* ctx) {
   if (!ctx->baseDecl().empty()) {
-    reportError(ctx->baseDecl(0), "BaseDecl is not supported.");
+    reportNotSupported(ctx->baseDecl(0), "BASE declarations are");
```

Comment:
If we can make `visitVector` also work for void types (with returning void instead of a vector), then this is a simple `visitVector`. We can talk about this later.

Reply:
I totally aggree that this would be the best approach. But of the top of my head I don't know how to integrate this with the current helpers. Lets talk about that in the meeting.